core/server/src/main/java/alluxio/worker/block/ClientRWLock.java

```diff
@@ -29,7 +29,7 @@
 public final class ClientRWLock implements ReadWriteLock {
   // TODO(bin): Make this const a configurable.
   /** Total number of permits. This value decides the max number of concurrent readers. */
-  private static final int MAX_AVAILABLE = 100;
+  private static final int MAX_AVAILABLE = 1000;
```

Comment:
Just for curiosity, do we know empirically that `1000` here is better than `100`?